cypress/support/pages/teams/pageTeams.js

```diff
@@ -0,0 +1,135 @@
+'use strict'
+
+class Teams {
+
+  static #addNewTeamButton ='[data-testid="add-team-btn"]'
+  static #teamName = '[data-testid="team_name"]'
+  static #teamDescription = '[data-testid="description_team"]'
+  static #teamChatActivateCheckBox = '[id="activateRC"]'
+  static #teamVideoActivationCheckBox = '[id="activateConf"]'
+  static #teamColourDropDown = '.sp-preview' //data-testid is to be added
+  static #teamCreateButton = '[data-testid ="create_team_btn"]'
+  static #teamNameOnOverviewPage = '[data-testid="title_of_an_element"]'
+  static #teamDescriptionOnOverviewPage = '[data-testid="body_of_element"]'
+  static #teamSettings = '[data-testid="team_settings"]'
+  static #teamEditOption = '[data-testid="edit_team_members"]' //data-testid is to be updated with correct name
+  static #teamSaveChages = '[data-testid="create_team_btn"]' //data-testid is to be updated with correct name
+  static #teamDeleteOption = '[data-testid="delete_team_members"]'
+  static #teamDeleteOnDailogBox = '[data-testid="btn-submit-action"]'
+
+
+
+  doNotSeeTeam (teamName) {
+    cy.get(Teams.#teamNameOnOverviewPage)
+      .contains(teamName)
+      .should('not.exist')
+  }
+
+  confirmDeleteOnDialogBox () {
+    cy.get(Teams.#teamDeleteOnDailogBox)
+      .click()
+  }
+
+  clickOnDeleteOption () {
+    cy.get(Teams.#teamDeleteOption)
+      .click()
+  }
+
+  clickOnSaveChangeButton () {
+    cy.get(Teams.#teamSaveChages)
+    .click()
+    .wait([
+      '@alerts_api',
+    ])
+    .then(interceptions => {
+      expect(interceptions.response.statusCode).to.equal(200)
+    })
+  }
+
+  clickOnEditOption () {
+    cy.get(Teams.#teamEditOption)
+      .click()
+      .wait([
+        '@alerts_api',
+      ])
+      .then(interceptions => {
+        expect(interceptions.response.statusCode).to.equal(200)
+      })
+  }
+
+  seeTeamEditPage () {
+      cy.url()
+        .should('include','/edit')
+  }
+
+  clickOnTeamSettings () {
+    cy.get(Teams.#teamSettings)
+      .eq(1)
+      .click()
+  }
+
+  clickOnAddTeam () {
+    cy.get(Teams.#addNewTeamButton)
+      .click()
+      .wait([
+        '@alerts_api',
+      ])
+      .then(interceptions => {
+        expect(interceptions.response.statusCode).to.equal(200)
+      })
+  }
+
+  seeTeamCreationPage () {
+    cy.url()
+      .should('include', '/teams/add')
+  }
+
+  enterTeamName (teamName) {
+    cy.get(Teams.#teamName)
+      .clear()
+      .type(teamName)
+  }
+
+  enterTeamDescription (desc) {
+    cy.get(Teams.#teamDescription)
+    .clear()
+    .type(desc)
+  }
+
+  seeChatActivationOption () {
+    cy.get(Teams.#teamChatActivateCheckBox)
+      .should('be.visible')
+  }
+
+  seeVideoActivationOption () {
+    cy.get(Teams.#teamVideoActivationCheckBox)
+      .should('be.visible')
+  }
+
+  selectTeamColour () {
```

Comment:
In this function, colour is not selected, only the drop down menu is shown. Also no assertion has been made to check whether the selected colour is the one we set in the step or not. As a result when the team is created the colour of the team we saw in the team overview is 'Black' instead of 'Red'

<img width="1728" alt="Screenshot 2022-11-22 at 14 43 19" src="https://user-images.githubusercontent.com/97288209/203329089-72a34968-6fb2-4682-a5b1-d12826676ff6.png">

Reply:
will be handled in a followup because of data-testid and app behavior.